svf/include/Graphs/ICFGNode.h

```diff
@@ -411,6 +453,23 @@ class CallICFGNode : public InterICFGNode
     SVFVar* vtabPtr;                /// virtual table pointer
     s32_t virtualFunIdx;            /// virtual function index of the virtual table(s) at a virtual call
     std::string funNameOfVcall;     /// the function name of this virtual call
+    const SVFVar* indFunPtr;
+
+    /// Constructor to create empty CallICFGNode (for SVFIRReader/deserialization)
```

Comment:
I have removed this line in my previous patch. Why it appears here again? Check your local commit and make sure you edit is on top of the latest mainstream